bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -25,6 +17,9 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
+import java.util.*;
```

Comment:
The great Google style guide above asks that we not use wildcards (*) in import statements and says that in return we shall receive happiness and honor.

i.e. Have an import statement for each class you're using from java.util instead of just one that imports everything =D

Reply:
Only import list